cernopendata/modules/fixtures/data/records/cms-tools-poet.json

```diff
@@ -0,0 +1,101 @@
+[
+  {
+    "abstract": {
+      "description": "<p>Physics Object Extractor Tool (POET) is an example C++ code (and its configuration in Python) to extract the physics object information from CMS Open data. These instructions are valid to work with the 2012 data in the AOD format. Snippets of this software can be used for physics analysis.</p>\n<p>The code has two packages. The PhysObjectExtractor package is the heart of the POET repository. It contains a collection of analyzers that extract information of different physics objects into a ROOT file called myoutput.root. They have been written separately for clarity and can be executed modularly using a single configuration file called poet_cfg.py. The BTagging package contains a B-tagging efficiency calculator.</p>"
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "CMS Open data group"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_created": [
+      "2012"
+    ],
+    "date_published": "2022",
+    "experiment": "CMS",
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "12501",
+    "run_period": [
+      "Run2012A",
+      "Run2012B",
+      "Run2012C",
+      "Run2012D"
+    ],
+    "source_code_repository": {
+      "url": "https://github.com/cms-opendata-analyses/PhysObjectExtractorTool/tree/2012"
+    },
+    "system_details": {
+      "description": "Use this software in the CMS Open Data VM environment (<a href=\"/record/252\">CMS VM Image for 2011, 2012 and 2015 CMS open data</a>) or in the corresponding <a href=\"/docs/cms-guide-docker\">Docker container</a>.",
+      "release": "CMSSW_5_3_32"
+    },
+    "title": "Physics object extractor tool for the CMS 2012 data",
+    "type": {
+      "primary": "Software",
+      "secondary": [
+        "Tool",
+        "Workflow"
```

Comment:
Will the POET repository on GitHub also contain some runnable workflow examples? If yes, then good. If not, then we should remove the "Workflow" and keep only "Tool".

(It's usually clearer if full analyses examples with output plots and everything are maintained separately from software tools, which would then give extra records such as https://opendata.cern.ch/record/12360... Will there be some separate example of this kind for POET too?)

Reply:
There's a simple [github workflow](https://github.com/cms-opendata-analyses/PhysObjectExtractorTool/blob/2012/.github/workflows/main.yml), but no it runs only on master. I'll update and check it with the branches

Reply:
BTW the `commands.sh` script there assumes `/mnt/vol` being available, and used as an input/output directory. It would be good to change it so that it can run from anywhere, i.e. also outside of the GitHub workflow command with the volume mounting.

Reply:
OK, I'll work on that. It should be possible to handle the GitHub specificities in the GitHub workflow file.